src/packages/sdk/src/engineCommands.ts

```diff
@@ -23,25 +22,26 @@ export interface ConfigMetaFormat {
  * See https://github.com/zeit/node-file-trace/issues/104
  */
 
-path.join(__dirname, '../query-engine-darwin')
-path.join(__dirname, '../introspection-engine-darwin')
-path.join(__dirname, '../prisma-fmt-darwin')
+// commented out, as these paths are different now with @prisma/engines
```

Comment:
Why not removed?

Reply:
Because they might be needed, so I want to keep the hints for now.
We'll see in the e2e tests if they're needed.